app/src/components/main/tooltips/hoverableTooltip/hoverableTooltip.scss

```diff
@@ -1,75 +1,149 @@
-.hover-rect {
-  position: relative;
+$ARROW_HEIGHT: 6px;
+$ARROW_WIDTH: 16px;
+
+.tooltip-trigger {
   display: inline-block;
-  width: 100%;
-  height: 100%;
-  vertical-align: top;
-
-  &:hover {
-    .hoverable-tooltip {
-      visibility: visible;
-      opacity: 1;
-    }
-  }
 }
 
-.hoverable-tooltip {
-  position: absolute;
-  visibility: hidden;
-  padding-top: 5px;
-  opacity: 0;
-  transition: visibility 0ms, opacity 300ms ease;
-  z-index: $Z-INDEX-TOOLTIP;
+/* stylelint-disable */
```

Comment:
I think it's better to turn off rules for specific strings instead of turning off linter for almost the whole file.

Reply:
It doesn't work with separate lines, because as I understood correctly, the linter runs on resulting css-file, and there are a lot of lines with attribute selector in it, so I sill get errors

Reply:
Try `/* stylelint-disable-line selector-max-attribute */`

Reply:
done with `/* stylelint-disable selector-max-attribute */`